cpp/CMakeLists.txt

```diff
@@ -365,6 +365,8 @@ add_library(cudf
             src/table/legacy/table.cpp
             src/strings/nvcategory_util.cpp
             src/join/legacy/joining.cu
+            src/join/join.cu
+            src/join/chuck_join.cu
```

Comment:
Plan to move chuck_join.cu into join.cu before a final merge, so this edit won't be necessary.

Reply:
BTW, if it makes sense to have a `semi_join.cu` rather than merging them, please do so -- more smaller compilation units increases parallel build efficiency.